netsync/manager.go

```diff
@@ -147,6 +147,25 @@ type peerSyncState struct {
 	requestedBlocks map[chainhash.Hash]struct{}
 }
 
+// limitAdd is a helper function for maps that require a maximum limit by
```

Comment:
Does anyone object to this func name? It's called `addAndLimitMap` in @dajohi code but I thought it was too verbose.

Reply:
no objection from me